ChromecastSampleApp/app/src/main/java/com/ooyala/sample/CastViewManager.java

```diff
@@ -21,17 +21,17 @@
     stateTextView = castView.findViewById(R.id.castStateTextView);
   }
 
-  void configureCastView(Video video) {
+  void configureCastView(String title, String description, String imageUrl) {
     final ImageView castBackgroundImage = castView.findViewById(R.id.castBackgroundImage);
 
     // Update the ImageView on a separate thread
-    new Thread(new UpdateImageViewRunnable(castBackgroundImage, video.getPromoImageURL(0, 0))).start();
+    new Thread(new UpdateImageViewRunnable(castBackgroundImage, imageUrl)).start();
```

Comment:
Why `new Thread`, I suppose you can just call `new UpdateImageViewRunnable().run()`